src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="search-form search-form_big"
+    data-qa="big"
+    >
+      <label
+      class="search-form__label search-form__label_big">
+        <input
+        type="text"
+        class="search-form__input search-form__input-big"
+        name="input_big"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress">
```

Comment:
Need to fix code style everywhere
```suggestion
        <input
          type="text"
          class="search-form__input search-form__input-big"
          name="input_big"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
```